src/Stratis.Features.FederatedPeg/Wallet/FederationWalletManager.cs

```diff
@@ -798,10 +830,7 @@ public bool RemoveTransientTransactions(uint256 depositId = null)
 
                 foreach ((Transaction transaction, IWithdrawal withdrawal) in this.FindWithdrawalTransactions(depositId))
                 {
-                    if (withdrawal.BlockNumber == 0)
```

Comment:
`if (withdrawal.BlockNumber == 0) ` is this done inside the method `this.RemoveTransaction`?

Reply:
This method has been renamed and no longer does that check before removing a withdrawal transaction. I've checked the rest of the code and when we call this methods the intention is to remove the withdrawal regardless of that field.